packages/runtime/src/useCases/consumption/attributes/SucceedRelationshipAttributeAndNotifyPeer.ts

```diff
@@ -41,10 +41,10 @@ export class SucceedRelationshipAttributeAndNotifyPeerUseCase extends UseCase<Su
     protected async executeInternal(request: SucceedRelationshipAttributeAndNotifyPeerRequest): Promise<Result<SucceedRelationshipAttributeAndNotifyPeerResponse>> {
         const predecessor = await this.attributeController.getLocalAttribute(CoreId.from(request.predecessorId));
         if (typeof predecessor === "undefined") {
-            return Result.fail(CoreErrors.attributes.predecessorDoesNotExist());
+            return Result.fail(CoreErrors.attributes.setPredecessorIdDoesNotMatchActualPredecessorId());
```

Comment:
There error here should rather be `predecessorDoesNotExist()`. Or am I missing something?

Reply:
No, I was just sloppy - fixed it